cli/pkg/github/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package github contains utilities to work with GitHub.
```

Comment:
What is the purpose of this file? 🙂

Reply:
Just to provide general package documentation ([godoc](https://blog.golang.org/godoc)). If it's not convenient for your team, I can delete it.

Reply:
I see now, thanks! We can leave this file as it is then.